allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -178,6 +186,38 @@ def init_context(
 
         return mp_ctx
 
+    def get_callback_classes(
```

Comment:
I wonder if it wouldn't be better to have this functionality in `ExperimentConfig`... Let's discuss it with @Lucaweihs when we have a chance.